shared/common-adapters/box.desktop.tsx

```diff
@@ -83,7 +83,7 @@ const _Box2 = (props: Box2Props, ref: React.Ref<HTMLDivElement>) => {
       ]
         .filter(Boolean)
         .join(' ')}
-      style={(style as unknown) as React.CSSProperties}
+      style={(Styles.collapseStyles([style]) as unknown) as React.CSSProperties}
```

Comment:
I thought we were trying to move toward calling `Styles.collapseStyles` before passing the style into components (cc @chrisnojima ?)

Reply:
i fixed another issue like this with nojima a week ago or so: https://github.com/keybase/client/pull/23025/files